src/api/app/views/layouts/webui2/webui.html.haml

```diff
@@ -9,6 +9,14 @@
     %title
       = "#{@pagetitle} - " if @pagetitle
       = home_title
+    %meta{ property: "og:title", content: meta_title}/
```

Comment:
these slashes at the end look too much

Reply:
Just as I thought:
bs/src/api/app/views/layouts/webui2/webui.html.haml:13: syntax error, unexpected tIDENTIFIER, expecting ')'

Reply:
Oh no... I missed a comma.

Should all these slashes be removed? I see in the HAML docs that meta tags are treated as being empty. So I guess they are a bit redundant.

Reply:
They are empty by default - so just remove them.